src/index.ts

```diff
@@ -15,6 +15,7 @@ let workersConnection: Redis | Cluster;
 if (config.redis.url) {
   connection = new IORedis(config.redis.url, {
     retryStrategy: () => 3000,
+    maxRetriesPerRequest: 20,
```

Comment:
This is strange because I think this is the default value (20) for maxRetriesPerRequest in IORedis.

Reply:
yes this was really odd. for some reason it crash on my computer when it is null. maybe itis my node version?

Reply:
No, I don't think so, what error do you get?